server/apps/distributed-app/docs/modules/ROOT/partials/RecipientRewriteTable.adoc

```diff
@@ -11,4 +11,9 @@ Example:
  <mailet match="All" class="RecipientRewriteTable">
    <errorProcessor>rrt-errors</errorProcessor>
  </mailet>
-....
\ No newline at end of file
+....
+
+The *rewriteSenderUponForward* option (fault to true) can be used to prevent senders to be rewritten upon forwards in the transport enveloppe
```

Comment:
```suggestion
The *rewriteSenderUponForward* option (default to true) can be used to prevent senders to be rewritten upon forwards in the transport envelope
```